pkgs/applications/science/biology/bftools/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, lib, pkgs, fetchurl, unzip, openjdk }:
+
+stdenv.mkDerivation rec {
+  name = "bftools-${version}";
+  version = "5.9.2";
+
+  src = fetchurl {
+    url = "http://downloads.openmicroscopy.org/bio-formats/${version}/artifacts/bftools.zip";
+    sha256 = "02x8p07z7js41kgiblkr9k8v9v4ka8rm7pg3f1vy9s5p6hmpfvnx";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ./* $out/
+    for script in $(find . -maxdepth 1 -perm -111 -type f); do
+      ln -s $out/$script $out/bin/$script
+    done
+  '';
+
+  postFixup = ''
+        for script in "$out"/bin/*; do
+          wrapProgram "$script" --prefix PATH : "${lib.makeBinPath [ openjdk ]}"
+        done
+      '';
+
+  buildInputs = [ unzip pkgs.makeWrapper ];
+
+  # phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
+
+  meta = with stdenv.lib; {
```

Comment:
Need to set `meta.platforms`, that's important for Hydra

Reply:
Should this be `platforms = platforms.linux ++ platforms.darwin;` or `platforms = platforms.all;` or `platforms = platforms.unix;` or something else? Presumably should be pretty portable thanks to java but have only tested NixOS and Ubuntu

Reply:
`platforms.all` is good. Your expression will only evaluate where `openjdk` evaluates.